P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -793,22 +793,23 @@ target="#CONA"/>).  The tag -->The
       </p>
 
       <p><soCalled>Characteristics</soCalled> or <soCalled>traits</soCalled> are typically independent of an individual's volition
-        or action and can be either physical, such as hair and eye colour; or cultural, such as ethnicity, caste, or faith.
+        or action and can be either physical, such as hair and eye colour; or cultural, such as ethnicity and caste.
         The distinction is not entirely straightforward: gender and sex can be regarded as culturally determined with physical manifestations, 
-        and the division of humanity into different <soCalled>races</soCalled>,
-        proposed by early anthropologists on the basis of physical characteristics such as skin colour, hair type,
-        and skull measurements, is now recognized as a social construct.</p>
+        interpreted based on the social hierarchies of a time and place. Also the division of humanity into different <soCalled>races</soCalled>,
+        formalized in the nineteenth century to organize a hierarchy of humanity on the basis of physical characteristics such as skin colour, hair type,
+        and skull measurements, is now recognized as a colonial social construct. When that construct defined individuals' identities beyond their volition, 
```

Comment:
I wonder if there should be a bibiographical reference

Reply:
@martinascholger There is so much we could cite, and it tends to be concentrated on specific colonized parts of the world. Looking for something quite general to cite, I wonder if this reference would work for the audience of the Guidelines? https://www.cambridge.org/core/books/abs/reproduction/colonialism-and-the-emergence-of-racial-theories/122DB95AC72DCB6CCDF6C7E034EFEC72

Reply:
@martinascholger I've added a citation now with this commit: https://github.com/TEIC/TEI/pull/2299/commits/d963460a4acc87b1105991db3814287c80c4ec1f 
 
Thanks for the suggestion! How does that look to you?